suite-native/module-accounts/src/components/AccountDetailGraph.tsx

```diff
@@ -22,19 +17,23 @@ type AccountDetailGraphProps = {
 
 export const AccountDetailGraph = ({ accountKey }: AccountDetailGraphProps) => {
     const fiatCurrency = useSelector(selectFiatCurrency);
-    const { graphPoints, error, isLoading, refetch, setHoursToHistory, hoursToHistory } =
-        useGraphForSingleAccount({
-            accountKey,
-            fiatCurrency: fiatCurrency.label,
-        });
-    const enhancedPoints = useMemo(
-        () => enhanceGraphPoints(graphPoints) as EnhancedGraphPointWithCryptoBalance[],
-        [graphPoints],
-    );
+    const {
+        graphPoints,
+        graphEvents,
+        error,
+        isLoading,
+        refetch,
+        setHoursToHistory,
+        hoursToHistory,
+    } = useGraphForSingleAccount({
+        accountKey,
+        fiatCurrency: fiatCurrency.label,
+    });
+
     const setSelectedPoint = useSetAtom(selectedPointAtom);
     const setReferencePoint = useSetAtom(referencePointAtom);
-    const lastPoint = A.last(enhancedPoints);
-    const firstPoint = A.head(enhancedPoints);
+    const lastPoint = A.last(graphPoints as FiatGraphPointWithCryptoBalance[]);
```

Comment:
Why is typecasting neccessary?

Reply:
It was necessary because `useGraphForAccounts` was returning points property as  `FiatGraphPoint[] | FiatGraphPointWithCryptoBalance[]`.  So inside of the `AccountDetailGraph` I needed to cast it to `FiatGraphPointWithCryptoBalance[]` to inform TS that the crypto balance is present. 

I solved this problem by creating a function overload of the `useGraphAccounts` function to return the correct type according to the use case. (`FiatGraphPointWithCryptoBalance[]` for detail graph and `FiatGraphPoint[]` for portfolio graph).


90955f4fc